src/test/java/com/listywave/acceptance/alarm/AlarmAcceptanceTest.java

```diff
@@ -0,0 +1,173 @@
+package com.listywave.acceptance.alarm;
+
+import static com.listywave.acceptance.alarm.AlarmAcceptanceTestHelper.알람_읽기_API_호출;
+import static com.listywave.acceptance.alarm.AlarmAcceptanceTestHelper.알람_조회_API_호출;
+import static com.listywave.acceptance.comment.CommentAcceptanceTestHelper.n개의_댓글_생성_요청;
+import static com.listywave.acceptance.comment.CommentAcceptanceTestHelper.댓글_저장_API_호출;
+import static com.listywave.acceptance.follow.FollowAcceptanceTestHelper.팔로우_요청;
+import static com.listywave.acceptance.list.ListAcceptanceTestHelper.리스트_저장_API_호출;
+import static com.listywave.acceptance.list.ListAcceptanceTestHelper.좋아하는_견종_TOP3_생성_요청_데이터;
+import static com.listywave.acceptance.reply.ReplyAcceptanceTestHelper.답글_등록_API_호출;
+import static com.listywave.list.fixture.ListFixture.가장_좋아하는_견종_TOP3;
+import static com.listywave.user.fixture.UserFixture.동호;
+import static com.listywave.user.fixture.UserFixture.유진;
+import static com.listywave.user.fixture.UserFixture.정수;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.listywave.acceptance.common.AcceptanceTest;
+import com.listywave.alarm.application.dto.AlarmListResponse;
+import com.listywave.alarm.application.dto.AlarmResponse;
+import com.listywave.list.application.domain.list.ListEntity;
+import com.listywave.list.presentation.dto.request.ListCreateRequest;
+import com.listywave.list.presentation.dto.request.ReplyCreateRequest;
+import com.listywave.list.presentation.dto.request.comment.CommentCreateRequest;
+import com.listywave.user.application.domain.User;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("알람 관련 인수테스트")
+public class AlarmAcceptanceTest extends AcceptanceTest {
+
+    @Test
+    void 리스트_생성시_콜라보레이터_지정_당한_사람들에게_알람이_간다() {
+        // given
+        User 정수 = 회원을_저장한다(정수());
+        User 동호 = 회원을_저장한다(동호());
+        User 유진 = 회원을_저장한다(유진());
+        String 정수_엑세스_토큰 = 액세스_토큰을_발급한다(정수);
+        String 동호_엑세스_토큰 = 액세스_토큰을_발급한다(동호);
+        String 유진_엑세스_토큰 = 액세스_토큰을_발급한다(유진);
+        ListCreateRequest listCreateRequest = 좋아하는_견종_TOP3_생성_요청_데이터(List.of(동호.getId(), 유진.getId()));
+        리스트_저장_API_호출(listCreateRequest, 정수_엑세스_토큰);
+
+        // when
+        ExtractableResponse<Response> response_동호 = 알람_조회_API_호출(동호_엑세스_토큰);
+        ExtractableResponse<Response> response_유진 = 알람_조회_API_호출(유진_엑세스_토큰);
+        AlarmListResponse result_동호 = response_동호.as(AlarmListResponse.class);
+        AlarmListResponse result_유진 = response_유진.as(AlarmListResponse.class);
```

Comment:
```suggestion
        AlarmListResponse 동호_알람_조회_결과 = 알람_조회_API_호출(동호_엑세스_토큰).as(AlarmListResponse.class);
        AlarmListResponse 유진_알람_조회_결과 = 알람_조회_API_호출(유진_엑세스_토큰).as(AlarmListResponse.class);
```

이렇게 간결하게 해도 될 것 같아요!

Reply:
오... 이처럼 굳이 나눌 필요 없겠네요 고마워요 😁😁